src/persistence/settings.cpp

```diff
@@ -1175,6 +1177,19 @@ void Settings::setCamVideoRes(QSize newValue)
     camVideoRes = newValue;
 }
 
+unsigned short Settings::getCamVideoFPS() const
+{
+    QMutexLocker locker{&bigLock};
```

Comment:
Don't do this in Getters/Setters. Rather put the mutex to the place, where you do call the method. Think what happens, if you call several getters like this in a row.

Is the mutex even required for reading?

Not sure if that matters here, but if called frequently from a loop, this *might* cause performance issues(?).